src/main/java/com/rakovets/course/java/core/practice/operators/Task04.java

```diff
@@ -33,8 +33,7 @@ public static void main(String[] args) {
      * @param milliseconds время (в миллисекундах) между срабатыванием датчика света и звука
      */
     static float getDistance(float milliseconds) {
-        //TODO
-        // Код, решающий задачу пишем ниже, при этом используя параметры метода
-        return 0.0f;
+
```

Comment:
"Violation: Useless blank lines
Solution: Remove blank lines"

Reply:
this fixed